treestats_paper.tex

```diff
@@ -1458,6 +1354,15 @@ \section*{Discussion}
 calculation of branch statistics would provide a summary method
 that is not affected by this potentially important confounding factor.
 
+Many genomic datasets only contain genotype information at a subset of sites
```

Comment:
This is good!

Reply:
yes!!! realized that on the bike ride in this morning, talking to andy.